### PR TITLE
Add FancyBox

### PR DESCRIPTION
**Pull request by craiglondon**
Testable at http://tealium.com/integrations/

### CODE REVIEW

**Comment by jvoisin** [comment]
Please fix the indentation :)

**Comment by craiglondon** [comment]
Indentation fixed, I had spaces instead of tabs.   I thought .editorconfig was supposed to take care of those issues... hmmmm.

File: src/apps.json [hunk 1]
```diff
@@ -2441,6 +2441,15 @@
 			"url": "(?:/ViewParametricSearch|ffsuggest\\.[a-z]htm)",
 			"website": "fact-finder.com"
 		},
+		"FancyBox": {
+			"cats": [
+				12
+			],
+			"icon": "FancyBox.png",
+			"implies": "jQuery",
+			"script": "jquery\\.fancybox\\.pack\\.js",
```
Comment (jvoisin) [inline src/apps.json]: What about `"jquery\\.fancybox\\.pack\\.js$"` instead?

**Comment by AliasIO** [comment]
Thanks!

**Comment by craiglondon** [comment]
@jvoisin This pattern isn't matching when I added the `$` as you requested.
`<script src="http://tealium.com/js/fancybox/jquery.fancybox.pack.js?v=2.1.5"></script>`
Please advise.

**Comment by jvoisin** [comment]
Maybe something like `"script": "jquery\\.fancybox\\.pack\\.js(?:v=([\d.]+))?$\\;version:\\1"`

**Pull request merged**